microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -614,8 +658,56 @@ When you select a folder to use as your Workspace, selecting any link to a file
 Any edits that you make in DevTools now change the file on the hard drive and sync live with Visual Studio Code. You can read about setting up your workspace in [Opening source files in Visual Studio Code](../sources/opening-sources-in-vscode.md).
 
 Status:
-*  This checkbox is present in Microsoft Edge Canary 113.
-*  This checkbox is present in Microsoft Edge Stable 111.
+*  This checkbox is present in Microsoft Edge Canary 114.
+*  This checkbox is present in Microsoft Edge Stable 112.
+
+
+<!-- ====================================================================== -->
+## Emulate Microsoft-recommended hardware and network performance presets
+
+_new as of Microsoft Edge 114_
+
+Emulates Microsoft-recommended hardware and network performance presets.
+<!-- todo: need text -->
+
+See also:
+*  todo
+
+Status:
+*  This checkbox is present in Microsoft Edge Canary 114.
+*  This checkbox is not present in Microsoft Edge Stable 112.
+
+
+<!-- ====================================================================== -->
+## Enable Crash Analyzer
+
+_new as of Microsoft Edge 114_
+
+Enables Crash Analyzer.
+<!-- todo: need text -->
+
+See also:
+*  todo
+
+Status:
+*  This checkbox is present in Microsoft Edge Canary 114.
+*  This checkbox is not present in Microsoft Edge Stable 112.
+
+
+<!-- ====================================================================== -->
+## Enable Adaptive CPU throttling
+
+_new as of Microsoft Edge 114_
+
+Enables Adaptive CPU throttling.
+<!-- todo: need text -->
```

Comment:
@captainbrosset @zoherghadyali need text for experiment checkbox "Enable Adaptive CPU throttling"

Is there a page to link to?  found:
https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/evaluate-performance/#simulate-a-mobile-cpu

Reply:
kept as-is